### PR TITLE
SetIsLuaChatBackendOverwritten in installer

### PR DESCRIPTION
**Pull request by spotco**

### CODE REVIEW

File: CoreScriptsRoot/Modules/Server/ServerChat/ChatServiceInstaller.lua [hunk 1]
```diff
@@ -23,8 +23,14 @@ end
 
 local function Install()
 	if (installDirectory:FindFirstChild(runnerScriptName)) then
+		pcall(function()
+			game:GetService("Chat"):SetIsLuaChatBackendOverwritten(true)
```
Comment (hockeychaos) [inline CoreScriptsRoot/Modules/Server/ServerChat/ChatServiceInstaller.lua]: do we still want this? says there is a conflict now

Comment (spotco) [inline CoreScriptsRoot/Modules/Server/ServerChat/ChatServiceInstaller.lua]: ya

Comment (spotco) [inline CoreScriptsRoot/Modules/Server/ServerChat/ChatServiceInstaller.lua]: ill merge it eventually not high priority

**Comment by spotco** [comment]
this is getting nuked

**Pull request closed without merging**